src/components/ListFormatControls/ListFormatControls.jsx

```diff
@@ -0,0 +1,26 @@
+import { useListContext } from "../../context/ListContext";
+import "./ListFormatControls.css";
+const ListFormatControls = () => {
+  const { delimiter, setDelimiter, wrapper, setWrapper } = useListContext();
+
+  return (
+    <div className="input-box-controls">
+      <label className="input-label">Wrapper: </label>
+      <input
+        className="input-text"
```

Comment:
wrap props with ```{}```

Reply:
What props? This is an HTML input element